pkg/command/root.go

```diff
@@ -101,3 +68,47 @@ func NewLogger(cfg *config.Config) log.Logger {
 		log.Color(cfg.Log.Color),
 	)
 }
+
+// ParseConfig load configuration for every extension
+// TODO: DRY this func, take Environment Prefix as parameter, as it is the only variable
+func ParseConfig(c *cli.Context, cfg *config.Config) error {
```

Comment:
Document why we need to call it in root and server?

Reply:
👍